docs/internal/config/Template-Builder.md

```diff
@@ -1,15 +1,140 @@
+
+**Note**: Form Templates must be in ‘DRAFT’ status before they can be edited. 
+Click the ‘Make Draft’ button in the config builder
+![General tab](../images/Template-builder3.png)
+
+In this tab is possible to change the status of this Template **version** to Make Available, Make Draft or Disable the application (for all end-users).
 ### Displayed options
 
-#### canApplicantMakeChangess
+| option             | template_column            | optional | default    | description                                                                                                                    |
+| ------------------ | -------------------------- | -------- | ---------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| Name               | name                       | no       | -          | Name of a **single application** displayed to users (in different areas of the App)                                            |
+| Name plural        | name_plural                | no       | -          | Name of the **group of applications** displayed to users (in different areas of the App)                                       |
+| Code               | code                       | no       | -          | Reference to application type. In case a new code is created this template version will be displayd as a new template type!    |
+| Serial Pattern     | serial_patern              | yes      | **?**      | The pattern to be used when generating a new application - used regular expression                                             |
+| Linear             | is_linear                  | yes      | **true**   | Determine if user is blocked from moving to the next page before filling all required fields                                   |
+| Interactive        | can_applicant_make_changes | yes      | **true**   | When this option is set to false it means no Reviewer can interact with Applicant, so option for "List of Questions" is hidden |
+| Category           | category                   | no       | -          | Used to group different template types in a category. The category visibility in areas of Conforma is also configurable        |
+| Filters            | -                          | yes      | -          | Selectable filters to display to users applications grouped by action to take in Dashboard                                     |
+| Start Message      | start_message              | yes      | Don't show | If defined will display the message on a page before creating the application                                                  |
+| Submission Message | submission_message         | yes      | **?**      | Show message to user on page after the application is submitted                                                                |
```

Comment:
These two should mention that value can be an evaluator expression

Reply:
Can it? I though it was just Markdown

Reply:
> Can it? I though it was just Markdown

Yes, definitely.

![Screenshot 2022-12-15 at 4 52 46 PM](https://user-images.githubusercontent.com/5456533/207768553-270b5b9f-3513-40a6-89b7-206fc22cd563.png)

Reply:
Thanks! Added. I think all has been addressed now (the Documentation in Wiki is also updated!)

Should we merged this one now @CarlosNZ ?